KrameWork/KW_ClassLoader.php

```diff
@@ -31,7 +31,7 @@ public static function loadClass($className)
 
 						$subClassPath = $classPath . DIRECTORY_SEPARATOR . $node;
 						if (is_dir($subClassPath))
-							array_unshift($subClassPath);
+							array_unshift($subClassPath, $subClassPath);
```

Comment:
@mortenn I'm even more confused now. $subClassPath is both the array and the value? Not sure if I'm missing something here. :D